### ISSUE TITLE
Misspellings in GUI (KeepNote 0.6.3)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#306]
```
I noticed a couple of minor typos, so I ran a spell checker on 
gettext/messages.pot:

msgid "Unable to dertermine note type."  (determine)
msgid "Swtch to previous tab"  (Switch)
msgid "Install Sucessful"  (Successful)
msgid "Uninstall Sucessful"  (Successful)
msgid "Use sysem tray icon"  (system)
```

Original issue reported on code.google.com by `csmayfi...@gmail.com` on 10 Jul 2010 at 9:13

**Comment by GoogleCodeExporter** [comment]
```
Thanks!
```

Original comment by `matt.rasmus@gmail.com` on 11 Jul 2010 at 2:03
- Added labels: **ui**

**Comment by GoogleCodeExporter** [comment]
```
This will be fixed in KeepNote 0.6.6.  Thanks.
```

Original comment by `matt.rasmus@gmail.com` on 7 Nov 2010 at 3:56
- Changed state: **Done**
